docs/user_docs/kubeblocks-for-redis/cluster-management/create-and-connect-a-redis-cluster.md

```diff
@@ -18,7 +18,7 @@ But for your better high-availability experience, KubeBlocks creates a Redis Pri
 
 * [Install kbcli](./../../installation/install-kbcli.md).
 * [Install KubeBlocks](./../../installation/install-kubeblocks.md).
-* Make sure the PostgreSQL addon is installed with `kbcli addon list`.
+* Make sure the redis addon is installed with `kbcli addon list`.
```

Comment:
Thank you for your contribution, using `Redis` with capitalized first letters would be better.

Reply:
OK. Fixed.